data/human/free worlds 4 epilogue.txt

```diff
@@ -44,8 +45,9 @@ mission "FW Epilogue: Ijs and Katya"
 				goto end
 			
 			label game
-			`	"Whoops," says Ijs, "now you've done it, there goes the fourth wall."`
-			`	Katya laughs. "Well, there's already lots beyond human space to see, and I'm sure you can find more things to do out there as well as closer to home. If you're so eager to explore more of the galaxy, <first>, or to discover what new adventures it has to offer, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it relies on people like you to bring it life."`
+			`	Your conversation is loudly interrupted by a resounding crack as the wall that Katya had worried about tumbles down, attracting a flurry of activity from the nearby construction workers.`
```

Comment:
This doesn't feel like an improvement to me, just heavy handed.

Reply:
So heavy, in fact, that it could be read too literally as him actually just complaining about the restaurant wall instead of it serving as a metaphor.

Far better I think to just add the initial more subtle lead-in hint of ongoing construction.